apps/total-typescript/schemas/documents/conceptPage.ts

```diff
@@ -0,0 +1,49 @@
+import {MdOutlineArticle} from 'react-icons/md'
+
+export default {
+  name: 'conceptPage',
```

Comment:
Looks similar to an article, but doesn't show up on the articles list and is hard-linked to a concept.